drivers/backup/portworx/portworx.go

```diff
@@ -302,6 +302,38 @@ func (p *portworx) WaitForClusterDeletion(
 	return nil
 }
 
+func (p *portworx) GetClusterUID(ctx context.Context, orgID string, clusterName string) (string, error) {
+	clusterEnumerateReq := &api.ClusterEnumerateRequest{
+		OrgId: orgID,
+	}
+	enumerateRsp, err := p.EnumerateCluster(ctx, clusterEnumerateReq)
+	if err != nil {
+		return "", err
+	}
+	for _, cluster := range enumerateRsp.GetClusters() {
+		if cluster.GetName() == clusterName {
+			return cluster.GetUid(), nil
+		}
+	}
+	return "", fmt.Errorf("cluster with name '%s' not found for org '%s'", clusterName, orgID)
+}
```

Comment:
We can simplify the function body to, right?
```
    clusterInspectReq := &api.ClusterInspectRequest{
		OrgId: orgID,
		Name: clusterName,
	}
	inspectRsp, err := p.InspectCluster(ctx, clusterInspectReq)
	if err != nil {
		return "", err
	}
	return inspectRsp.Cluster.GetUid(), nil
```

Reply:
It is advised to use ClusterEnumerateRequest as it aligns with the legacy code. Functions like GetBackupUID, GetBackupName, GetClusterUID, GetClusterName, GetClusterStatus, GetBackupLocation, and other similar functions relies on EnumerateRequest.